### ISSUE TITLE
Cache hit/miss should be visible in hook context

### ISSUE THREAD

**Issue by tvld** [feathersjs-ecosystem/feathers-hooks-common#614]
I might just not be able to find it, but I think that in a hook, there is no way to check if results come from the cache or not?

For debugging and profiling, it is valuable to know this...

**Comment by fratzinger** [comment]
You're right. I think this is not possible atm. Are you up to add a PR for this?

**Comment by tvld** [comment]
I am not yet really up to the task, I am afraid... Hopefully I can learn more and be back with a Pull Request ))
